src/lib/convert_types.go

```diff
@@ -0,0 +1,75 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lib
```

Comment:
I recall there are some similiar methods in the configuration code base, you can confirm with @stonezdj

Reply:
Do you mean the `Type` interface or `ConfigureValue` in the `common/config/metadata`? I don't think it does the same thing as the functions in this file.

The functions in this file are some helpers to convert between ptr and value for some native types.

Maybe I need to rename `ToBool` to `ToBoolValue`.

Reply:
`Bool`, `Int64` and `String` has been removed, `BoolValue`, `ToBool` not need to rename.